### DISCUSSION TITLE
List Clients - Issue with filterable fields

### DISCUSSION THREAD

**Discussion by dlondero** [fattureincloud/api#16] [Q&A] [answered]
Hi there, when using `List Clients` endpoint it is possible to pass a `q` param in the querystring to filter results. Which fields are available for filtering in this way?

Did some tests successfully on `vat_number` but then trying with `tax_code` had no luck.

**Comment by valmoz** [comment, marked as answer]
Hi @dlondero 
You can find the list of available fields for filtering [at this link](https://developers.fattureincloud.it/docs/basics/additional-capabilities/filter-results#-filterable-fields).

"tax_code" is listed as filterable, can you send us the query you're inserting into "q"? (please, use fake data and not true values)

Mauro

**Reply by valmoz** [reply]
Hi @dlondero I'm going to set this discussion as answered since we didn't get an answer from you in the last 5 days.
Feel free to reopen it if you still need help.

Thanks
Mauro

**Reply by dlondero** [reply]
Hi @valmoz, I did some more testing and I can confirm "tax_code" works as expected. I think I missed the quotes in my previous tests. Thanks for the support!
